package.json

```diff
@@ -108,13 +109,13 @@
     "env:stop": "./bin/local-env/stop.sh",
     "env:reset-site": "./bin/local-env/install-wordpress.sh --reset-site",
     "lint": "npm-run-all --parallel lint:*",
-    "lint:css": "wp-scripts lint-style",
+    "lint:css": "git diff develop --diff-filter=d --name-only | grep -E 'css$' | xargs wp-scripts lint-style",
```

Comment:
`grep -E 'css$'` is to accomodate possible `.scss` files, though I don't think there are any now